profiles/schema.py

```diff
@@ -904,9 +904,13 @@ def mutate(parent, info, input):
                 address_model = address_type["model"]
```

Comment:
Good change. Although I would do the address thing something like this:

```python
        for address_type in address_types:
            address_input = address_type["input"]
            if not address_input:
                continue

            address_name = address_type["name"]
            address_model = address_type["model"]
            try:
                address = getattr(information, address_name)
            except address_model.DoesNotExist:
                address = address_model(verified_personal_information=information)
                
            for field, value in address_input.items():
                setattr(address, field, value)

            if not address.is_empty():
                address.save()
            else:
                if address.id:
                    address.delete()
```

Reply:
That indeed looks more straightforward.